src/sentry/integrations/bitbucket/integration.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+from sentry.integrations import IntegrationProvider, IntegrationMetadata
+from sentry.pipeline import NestedPipelineView
+from sentry.identity.pipeline import IdentityProviderPipeline
+from django.utils.translation import ugettext_lazy as _
+DESCRIPTION = """
+BitBucket for Sentry.io
+"""
+metadata = IntegrationMetadata(
+    description=DESCRIPTION.strip(),
+    author='The Sentry Team',
+    noun=_('Repository'),
```

Comment:
do you have to install on each repository? or should this be something that refers to a bitbucket org or account?

Reply:
@macqueen Bitbucket doesn't appear to have organizations. You have repositories, teams, and users. But many teams can access the same repositories; it's not a 1-to-1. I can do the bitbucket.org account instead.

Reply:
Here's a list to the api reference I'm looking at. https://developer.atlassian.com/bitbucket/api/2/reference/resource/users let me know if I'm missing something